python/SciPass.py

```diff
@@ -922,14 +947,13 @@ def addPrefix(self, dpid=None, domain_name=None, group_id=None, prefix=None, pri
                                               idle_timeout = 0,
                                               hard_timeout = 0,
                                               priority     = priority)
-
-
+      
       if(prefix._version != 4):
-        header = {"dl_type": 34525}
+        header = {"dl_type": 34525,
```

Comment:
can we remove this if? seems like this is taken care of based on the prefix type later on?